exercises/practice/pig-latin/src/main/java/PigLatinTranslator.java

```diff
@@ -1,10 +1,10 @@
-/*
+import java.lang.String;
+public class PigLatinTranslator {
+    public PigLatinTranslator() {
+        throw new UnsupportedOperationException("Please implement the PigLatinTranslator constructor.");
+    }
 
-Since this exercise has a difficulty of > 4 it doesn't come
-with any starter implementation.
-This is so that you get to practice creating classes and methods
-which is an important part of programming in Java.
-
-Please remove this comment when submitting your solution.
-
-*/
+    public String translate(String text) {
```

Comment:
I would prefer the method parameter to be named `sentence` like in the example implementation. It gives the student a small hint in terms of what to expect in terms of input.